menu/EditableMenuEntry.js

```diff
@@ -0,0 +1,150 @@
+export class EditableMenuEntry {
+  constructor(dishAndPriceArray, menuContainer, i) {
+    this.dishAndPriceArray = dishAndPriceArray;
+    this.menuContainer = menuContainer;
+    this.i = i;
+    this.row = null;
+    this.editForm = null;
+    this.editEntryWrapper = null;
+    this.dishNameEditInput = null;
+    this.priceInEUREditInput = null;
+    this.price = null;
+    this.dishName = null;
+    this.createMenuEntry();
+  }
+  createMenuEntry() {
+    this.row = document.createElement("div");
+    this.row.classList.add("menu-row");
+    this.createDishNameP();
+    this.createPriceWithCurrencyDiv();
+    this.createButtonsInWrapper();
+    this.menuContainer.append(this.row);
+  }
+
+  createDishNameP() {
```

Comment:
```suggestion
  createDishNameParagraph() {
```
Let's use the full names, so it is clear.